### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by eftekhari-mhs**
sharing the mcf,tmc, and csv files

### CODE REVIEW

File: scripts/open_covid_data/open_covid_19_data.mcf [hunk 1]
```diff
@@ -0,0 +1,972 @@
...
+populationType: dcs:MedicalTest
+medicalTestingMethod: dcs:COVID19PCRTest
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_Person_COVID_19_Tested
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
```
Comment (tjann) [inline scripts/open_covid_data/open_covid_19_data.mcf:640]: Do these two have the wrong pop type? Please tag me for a second review once this is sorted out.

File: scripts/open_covid_data/open_covid_19_data.mcf [hunk 2]
```diff
@@ -0,0 +1,972 @@
...
+
+Node: dcid:CumulativeCount_MedicalTest_COVID_19_PCR
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+medicalTestingMethod: dcs:COVID19PCRTest
+statType: dcs:measuredValue
```
Comment (tjann) [inline scripts/open_covid_data/open_covid_19_data.mcf:648]: incremental or cumulative? Please tag me to take a second look when fixed.

File: scripts/open_covid_data/open_covid_19_data.mcf [hunk 3]
```diff
@@ -0,0 +1,972 @@
...
+
+Node: dcid:CumulativeCount_MedicalTest_COVID_19
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
```
Comment (tjann) [inline scripts/open_covid_data/open_covid_19_data.mcf:655]: ditto, inconsistent cumu/incr

File: scripts/open_covid_data/open_covid_19_data.mcf [hunk 4]
```diff
@@ -0,0 +1,972 @@
...
+spatialCoverage: dcs:AllAdministrativeAreas
+
+Node: dcid:PolicyExtent_Legislation_COVID19Pandemic_GovernmentOrganization_SchoolClosure_Unknown
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+legislationPassedBy: schema:GovernmentOrganization
+category: dcs:COVID19Pandemic
+policyType: dcs:SchoolClosure
+measuredProperty: dcs:policyExtent
+spatialCoverage: dcs:Unknown
```
Comment (tjann) [inline scripts/open_covid_data/open_covid_19_data.mcf:690]: Now realizing this causes the StatVar ID to be confusion. Let's change this enum to SpatialCoverageUnknown

File: scripts/open_covid_data/open_covid_19_data.mcf (hunk 1 above)
Comment (eftekhari-mhs) [reply scripts/open_covid_data/open_covid_19_data.mcf]: @tjann Can you please take a second look?

Comment (tjann) [reply scripts/open_covid_data/open_covid_19_data.mcf]: I still see "dcid:CumulativeCount_Person_COVID_19_Tested" and "populationType: dcs:MedicalTest" -- is that what you see too?

Comment (tjann) [reply scripts/open_covid_data/open_covid_19_data.mcf]: @eftekhari-mhs

Comment (eftekhari-mhs) [reply scripts/open_covid_data/open_covid_19_data.mcf]: no it is changed to person

File: scripts/open_covid_data/open_covid_19_data.mcf [hunk 5]
```diff
@@ -0,0 +1,964 @@
...
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+populationType: dcs:Person
+healthPrevention: dcs:COVID19PCRTest
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_Person_COVID19Tested
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
```
[the remark this answers is not in the document]
Comment (eftekhari-mhs) [reply scripts/open_covid_data/open_covid_19_data.mcf:629]: @tjann We don't need this node, I deleted it. It's not in the tmcf file and Jay said "The cc-by dataset does not have that unit".

[the remark this answers is not in the document]
Comment (tjann) [reply scripts/open_covid_data/open_covid_19_data.mcf:629]: OK

**Pull request closed without merging**